### PR TITLE
Pull request to approve rspec test for Player and TicTacToe 

### PR DESCRIPTION
**Pull request by fedgut**

### CODE REVIEW

File: spec/tic_tac_toe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  end
+
+  describe '#set' do
+    it 'should set the choosen number into the board' do
+      game = TicTacToe.new([Player.new(1, 'Eduardo'), Player.new(2, 'Tiago')])
+      board_ = (1..9).to_a
+      board_[4] = 'O'
+      expect { game.set(5) }.to(change { game.board }.from((1..9).to_a).to(board_))
+    end
+  end
```
Comment (bolah2009) [inline spec/tic_tac_toe_spec.rb:23]: This test is failing.

```ruby
Failures:

  1) TicTacToe#set should set the choosen number into the board
     Failure/Error: expect { game.set(5) }.to(change { game.board }.from((1..9).to_a).to(board_))
       expected `game.board` to have changed to [1, 2, 3, 4, "O", 6, 7, 8, 9], but is now [1, 2, 3, 4, "0", 6, 7, 8, 9]
     # ./spec/tic_tac_toe_spec.rb:21:in `block (3 levels) in <top (required)>'

Finished in 0.04224 seconds (files took 0.46855 seconds to load)
9 examples, 1 failure

Failed examples:

rspec ./spec/tic_tac_toe_spec.rb:17 # TicTacToe#set should set the choosen number into the board
```

File: bin/main.rb [hunk 2]
```diff
@@ -1,85 +1,102 @@
...
-    # TODO: validate (min 3 letters)
-    players[i] = Player.new(i, name)
+  until accept_players == true
+    2.times do |i|
+      accept_name = nil
+      until accept_name == true
+        print "Please tell us player #{i + 1} name: "
+        name = gets.chomp
+        name = name.gsub(/\d/, "")   
+        name = name.gsub(/\W/, "")
```
Comment (bolah2009) [inline bin/main.rb]: All these lines could be in a single line, right? :thinking: 

```ruby
name = gets.chomp.gsub(/\d|\W/, "")
```

File: bin/main.rb [hunk 3]
```diff
@@ -1,85 +1,102 @@
...
-def ask_names
+def get_player_names
+  accept_players = nil
   players = []
-  2.times do |i|
-    print "Please tell us player #{i + 1} name: "
-    name = gets.chomp
-    # TODO: validate (min 3 letters)
-    players[i] = Player.new(i, name)
+  until accept_players == true
```
Comment (bolah2009) [inline bin/main.rb]: using `until accept_players` should work since `until` is excepting true/false statement.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi Team,

Please don't submit another CRR ( code review request) if other milestones are not completed yet.

Let's make sure #4  and #5 are merged to `develop` branch and `feature/spec` is rebased and in sync with the `develop` branch before submitting another CRR.

Fix linter errors/warnings

#### :recycle: Changes required.


-----

### Let's make this Pull Request look more professional.
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [REQUIRED] Use meaningful names for  Pull Requests.
  - [REQUIRED] Add a short summary of changes in the Pull Request description.

#### Rename PR
  rename PR
-----
#### Update PR description
  update-PR-description-html

**Comment by fedgut** [comment]
> Hi Team,
> 
> Please don't submit another CRR ( code review request) if other milestones are not completed yet.
> 
Heya! we understand the idea behind not opening a code review request until the prev. milestones are done, but if code review takes more than 24 hours, (like 48 hours in this case) we can't move forward if we don't submit further code review request for more work done in the project.

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap:  You did it :smile: 

**The project is approved** :+1: 

Sorry that your review took more than 24 hours. The next time, please, write us on Slack **fulltime students** channels. Really, sorry.

Good luck with the next project.

**Pull request merged**